.circleci/discogapic_compute_build.gradle

```diff
@@ -23,11 +23,10 @@ compileJava.options.encoding = 'UTF-8'
 javadoc.options.encoding = 'UTF-8'
 
 dependencies {
-    compile 'com.google.api:gax:1.29.0'
-    testCompile 'com.google.api:gax:1.29.0:testlib'
-    compile 'com.google.api:gax-httpjson:0.51.0'
-    testCompile 'com.google.api:gax-httpjson:0.51.0:testlib'
-    testCompile 'io.grpc:grpc-netty-shaded:1.10.1'
```

Comment:
Why no more netty-shaded? Was this always unnecessary?

Reply:
Right.